src/common/localization/en-us.ts

```diff
@@ -57,6 +58,22 @@ export const english: IAppStrings = {
         messages: {
             deleteSuccess: "Successfully deleted ${project.name}",
         },
+        homeProjectView: {
+            title: "Train and use a model with labels"
+        },
+        prebuiltPredict: {
+            title: "Use prebuilt model to get data",
+            description: "Start with a pre-built model for forms link receipts or business cards. Submit your data and get result right away."
```

Comment:
We need to be consistent in wording and use either "pre-built" or "prebuilt", but not both everywhere.